content/issues/2021-06-28-vacation.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Summer is here ☀️"
+date: 2021-06-25T10:55:00+02:00
+resolved: false
+# resolvedWhen: 2021-06-04 09:10:40+00:00
+severity: notice
+affected:
+  - API
+  - Workers
+  - Dashboard
```

Comment:
This makes them shown in a Maintenance mode, but they most likely won't be affected.

```suggestion
```

Reply:
thanks! didn't realize it works like this